app/package.json

```diff
@@ -30,7 +31,9 @@
   },
   "devDependencies": {
     "belly-button": "4.x.x",
+    "dotenv": "^5.0.1",
```

Comment:
we don't have any rules about this, but tend to use the `5.0.x` style when its a 3rd party module and `5.x.x` when we control it